mlos_bench/mlos_bench/config/README.md

```diff
@@ -8,6 +8,13 @@ In general the `config` directory layout follows that of the `mlos_bench` module
 
 Full end-to-end examples are provided in the [`cli`](./cli/) directory, and typically and make use of the root [`CompositeEnvironments`](./environments/root/) to combine multiple [`Environments`](./environments/), also referencing [`Services`](./services/), [`Storage`](./storage/), and [`Optimizer`](./optimizers/) configs, into a single [`mlos_bench`](../run.py) run.
 
+## Config parameters
+
+An `Environment` configuration can have two sections, `const_args` and `tunable_params`.
+At runtime, at each trial the data from both sections will be merged and passed to the environment as a single dictionary of key/value pairs.
+The difference between `const_args` and `tunable_params` is that the former values stay constant across all trials, whereas `tunable_params` values can change from one trial to the next (e.g., when the optimizer proposes a new configuration).
```

Comment:
Not sure this is entirely true either.
`const_args` can read from environment variables or from a value further up in the stack of configs, so they're not exactly "constant", but more so "defaults" for `required_args`